cmd/agentbasedinstaller/register.go

```diff
@@ -87,7 +88,18 @@ func RegisterCluster(ctx context.Context, log *log.Logger, bmInventory *client.A
 	log.Info("releaseImage: " + releaseImage)
 	log.Infof("releaseImage version %s cpuarch %s", releaseImageVersion, releaseImageCPUArch)
 
-	clusterParams := controllers.CreateClusterParams(&cd, &aci, pullSecret, releaseImageVersion, releaseImageCPUArch, nil)
+	var operatorInfo []*models.OperatorCreateParams
+	fileInfo, _ := os.Stat(operatorInstallPath)
```

Comment:
Is it OK to ignore all errors here? For example, what happens if the file exists but in a directory that doesn't have execution permission for the current user? Would it make sense to check the error and report it if it is something other than `os.ErrNotExist`?

Reply:
Good idea. Updated to add check.